modules/tinymce/src/core/test/ts/browser/EditorAutoFocusTest.ts

```diff
@@ -0,0 +1,63 @@
+import { describe, it, before, afterEach } from '@ephox/bedrock-client';
+import { Insert, SugarBody, SugarElement } from '@ephox/sugar';
+import { TinyHooks } from '@ephox/wrap-mcagar';
+import { assert } from 'chai';
+
+import Editor from 'tinymce/core/api/Editor';
+import { EditorManager } from 'tinymce/core/api/PublicApi';
+
+const isInViewport = (editor: Editor) => {
+  const containerRect = editor.getContentAreaContainer().getBoundingClientRect();
+  const innerRect = editor.selection.getNode().getBoundingClientRect();
+  const top = containerRect.top + innerRect.top + 1;
+  return top > 0 && top + innerRect.height < window.innerHeight;
+};
+
+const setupEditorAutoFocus = (id: string) => {
+  return new Promise((resolve) => {
+    EditorManager.init({
+      selector: 'textarea',
+      base_url: '/project/tinymce/js/tinymce/',
+      toolbar: '',
+      menubar: false,
+      statusbar: false,
```

Comment:
`toolbar: ''` is not the correct way to disable the toolbar, you should be using `toolbar: false`. In saying that though, as mentioned elsewhere we need these enabled to test they are still in view.